packages/generate/src/syntax/casting.ts

```diff
@@ -113,11 +113,11 @@ type getAssignmentLiteral<
   ?
       | TsType
       | (Set["__cardinality__"] extends Cardinality.Many
-          ? TsType[]
+          ? readonly TsType[]
```

Comment:
Can you add a simple test to https://github.com/edgedb/edgedb-js/blob/master/integration-tests/lts/casts.test.ts to cover both `readonly` and mutable arrays?